src/Compilers/CSharp/Portable/Syntax/NonRecursiveRewriter.cs

```diff
@@ -0,0 +1,287 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
```

Comment:
Actually they are sorted.I can move Microsoft.CodeAnalysis.CSharp.Syntax after System.Linq if you want but then they won't be sorted...